### DISCUSSION TITLE
求助！！！Doris 使用SparkLoad时提交任务失败

### DISCUSSION THREAD

**Discussion by fdf1779** [apache/doris#27038] [A - General / Q&A]
Hadoop 版本3.1.1.x
Spark 版本 2.3.2.x
Doris 版本2.0.2 
使用SparkLoad 同步Hive分区表的时候，Doris 已经把 归档文件和 信息上传到HDFS /doris_prd_data 下面了
但是任务show load 查看任务状态时任务呗cancel 了，并且fe.warn 的log中只有 cancel的原因是ETL_SUBMIT_FAIL;
WARN (pending-load-task-scheduler-pool-7|760) [LoadJob.unprotectedExecuteCancel():660] LOAD_JOB=1075010, transaction_id={2118006}, error_msg={Failed to execute load with error: }
这条警告信息。其他的报错信息就没有了。
有没有遇到过的朋友有什么线索可以提供一下

**Comment by feng-kui** [comment]
我也有一样的问题，等待有缘人解惑

**Reply by fdf1779** [reply]
我的解决了，重新打了FE的包，主要是改了maven依赖，用的Master 分支的，然后改了一下Spark-Dpp DataTIMEV2的 时间解析的算法。

**Reply by feng-kui** [reply]
那就是还是原Doris代码的问题么
